lib/internal/async_hooks.js

```diff
@@ -316,27 +316,31 @@ function defaultTriggerAsyncIdScope(triggerAsyncId, block, ...args) {
   }
 }
 
+function hasHooks(key) {
+  return async_hook_fields[key] > 0;
+}
```

Comment:
What if i want to test a case where `async_hook_fields[key] < 0` or `async_hook_fields[key] == 0` because negation of this will only give me something related `async_hook_fields[key] <= 0`. Is this abstraction covers every case. From this i want to convey that use this abstraction for a single purpose which is solid.

Like this `!hasHooks(kInit)` conveys very different meanings.

Reply:
`async_hook_fields` is a `Uint32Array`, its entries are never negative. Maybe that clears this up a bit?

Reply:
Let's explain this problem in conjunction with `sync_hooks`. `async_hook_fields [key]` is obviously used to determine whether the corresponding `key` has `Active Hooks`. In this case, we only need to satisfy two situations, one is that `> 0` exists `Active Hooks`, and the other is `<= 0`, there is `no Active Hooks`

Reply:
> async_hook_fields is a Uint32Array, its entries are never negative. Maybe that clears this up a bit?

yes, that's an explanation.

Reply:
Cool, makes sense. 👍